src/server/api/auth.ts

```diff
@@ -84,6 +85,69 @@ authRouter.post("/register", async (req, res, next) => {
     }
 })
 
+// POST /api/auth/login
+authRouter.post("/login", async (req, res, next) => {
+  try {
+    const { email, password } = req.body;
+
+    const user = await prisma.user.findUniqueOrThrow({
+        where: {
+          email: email,
+        },
+      });
+  
+
+    // if (user) {
+    bcrypt.compare(
+      password,
+      user.password,
+      async function (err: Error | undefined, result: boolean) {
+        if (err) {
+          next(err)
+        } else if (result) {
```

Comment:
Something like this was set up in Zoop:

```suggestion
        if (err) {
        console.error("Error in bcrypt.compare:", err);
        res.status(500);
          next({name: "InternalServerError"})
        } else if (result) {
```